app/views/assignments/new.html.erb

```diff
@@ -75,59 +75,49 @@
       </div>
 
       <!-- Rubric Card -->
-      <div class="bg-white rounded-lg shadow-md p-6 transition-all hover:shadow-lg" 
-           data-controller="rubric-toggle"
-           data-rubric-toggle-switch-active-class="bg-blue-600"
-           data-rubric-toggle-switch-inactive-class="bg-gray-200"
-           data-rubric-toggle-knob-active-class="translate-x-5"
-           data-rubric-toggle-knob-inactive-class="translate-x-0"
-           data-rubric-toggle-textarea-disabled-class="bg-gray-100"
-           data-rubric-toggle-text-light-class="text-gray-500"
-           data-rubric-toggle-hidden-class="hidden">
+      <div class="bg-white rounded-lg shadow-md p-6 transition-all hover:shadow-lg">
         <div class="flex items-center mb-4">
           <%= render "shared/icons/clipboard_list", aria_hidden: true, focusable: false, class: "h-5 w-5 mr-2 text-blue-500" %>
           <h4 class="text-lg font-medium leading-6 text-gray-900">Rubric</h4>
         </div>
-        <p class="text-sm text-gray-500 mb-4">Choose how to provide the grading rubric. You can generate one using AI based on the assignment details or paste your own.</p>
+        <p class="text-sm text-gray-500 mb-4">GradeBot will generate an AI rubric based on your assignment details.</p>
 
         <div class="flex items-center justify-between mb-4">
           <span class="flex grow flex-col">
             <%# Generate Label %>
-            <span class="text-sm font-medium text-gray-900 flex items-center gap-1" id="rubric-generate-label" data-rubric-toggle-target="generateLabel">
```

Comment:
I'd like to create a markdown file called `docs/rubric_toggle.md` include full documentation on how to re-enable this feature. It should include all the necessary data labels fully documented.

Reply:
Also add to the docs/decision_log.md that we decided to remove the ability to paste a rubric to avoid confusion with initial user testing. Link to the rubric_toggle.md explaining that this doc shows how to re-enable.